src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -174,6 +174,8 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                                 telemetry.TelemetryEvent[TotalUniquePackagesCount] = cacheFile.ExpectedPackageFilePaths?.Count ?? -1;
                                 telemetry.TelemetryEvent[NewPackagesInstalledCount] = 0;
 
+                                await UpdateLastAccessTime(cacheFile);
```

Comment:
Same here, probably best if included in the measured restore time.

Reply:
Done